package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@primer/components",
-  "version": "28.2.3",
+  "version": "28.2.5",
```

Comment:
this just pulled the version number forward to the package-lock, since it appears it wasn't updated when the last version numbers were bumped